src/main/java/domain/LottoPrize.java

```diff
@@ -27,10 +30,12 @@ public static LottoPrize of(int matchingNumber, boolean containsBonus){
             return containsBonus ? SECOND_PRIZE : THIRD_PRIZE;
         }
 
-        for(LottoPrize lottoPrize : LottoPrize.values()) {
-            if(lottoPrize.getMatchingNumber() == matchingNumber) {
-                return lottoPrize;
-            }
+        Optional<LottoPrize> optionalLottoPrize = Arrays.stream(LottoPrize.values())
+                .filter(lottoPrize -> lottoPrize.getMatchingNumber() == matchingNumber)
+                .findFirst();
+
+        if(optionalLottoPrize.isPresent()) {
+            return optionalLottoPrize.get();
         }
 
         return NOTHING;
```

Comment:
```suggestion
       return optionalLottoPrize.orElse(NOTHING);
```
요렇게 표현해볼 수도 있을거 같아요! 😄